client/web/src/repo/RevisionsPopover.tsx

```diff
@@ -224,18 +228,30 @@ export const RevisionsPopover: React.FunctionComponent<Props> = props => {
 
     return (
         <Tabs defaultIndex={tabIndex} className="revisions-popover" onChange={handleTabsChange}>
-            <div className="tablist-wrapper w-100 align-items-center">
+            <div className="tablist-wrapper flex-1">
                 <TabList>
                     {TABS.map(({ label, id }) => (
-                        <Tab className="d-flex flex-1 justify-content-around" key={id} data-test-tab={id}>
-                            {label}
+                        <Tab className="tablist-wrapper__full-width" key={id} data-test-tab={id}>
+                            <span className="tablist-label">{label}</span>
                         </Tab>
                     ))}
                 </TabList>
+                {isRedesignEnabled && (
+                    <button
+                        type="button"
+                        // TODO: add onClick funcitonality
```

Comment:
Can you please open a ticket and link it in the TODO here?

```suggestion
                        // TODO: add onClick functionality
```

Reply:
Actually, given that this button is not functional and there's very little code here, it's probably better to remove the code here from this PR, and add it only when you implement the onClick functionality. Overall I'm in favour of working incrementally, and I believe that it's totally ok to have incomplete code that is inactive, but it doesn't seem worth it in this case.